src/utils/index.js

```diff
@@ -1,6 +1,7 @@
 export const makeConstant = (branchOfState, actionType) => `${branchOfState}-${actionType}`;
 
 export const standardizePhoneNumber = (hotlineNumber) => {
+  if (!hotlineNumber) return ''
   // returns the input as is if the field value includes notes (ie, multiple phone numbers)
   if (hotlineNumber.length > 14) return hotlineNumber
```

Comment:
i'm going to merge this in to fix this bug, but I prefer brackets even in one line returns. I have seen it cause issues when people less familiar with javascript come into a code base and accidentally put a new line that changes the behavior